examples/tutorial_derive/04_04_custom.rs

```diff
@@ -75,7 +75,7 @@ fn main() {
         let input = cli
             .input_file
             .as_deref()
-            .or_else(|| cli.spec_in.as_deref())
+            .or(cli.spec_in.as_deref())
```

Comment:
this is correct, since `as_deref` is now `const`. The clippy version used in the github workflow is too old to recognise this, and throwing a false positive.

Reply:
or, more correctly, this method probably isn't `const` when using the older rust version...

still, i think it's better to *target* the head and *support* the MSRV, than the other way around